saas_demo/models/saas_operator.py

```diff
@@ -62,7 +62,12 @@ def update_repos(self):
     def update_odoo(self):
         """Fetch and checkout Repository"""
         if self.is_local():
-            git(root_odoo_path(), ['pull', 'origin'])
+            test = self.env['ir.config_parameter'].get_param('test')
```

Comment:
choose more specific name for the variable

Reply:
test_run or is_test?

Reply:
I mean name for config param